src/Enums/ticketStates.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
To match the PSR-4 autoloader convention - this file should be name exactly the same as the class name, including its case (`TicketStates`).

Reply:
Good catch! I'll make the change.